polyfill.js

```diff
@@ -0,0 +1,13 @@
+if (typeof window.Map === 'undefined') {
+	window.Map = require('can-cid/map/map');
```

Comment:
Isn't Justin trying to remove can-cid from everything?

I think this is fine and not worth not approving this, but I wonder if we need to get our polyfills from somewhere else.